osu.Game/Screens/OnlinePlay/Multiplayer/MultiplayerLoungeSubScreen.cs

```diff
@@ -39,6 +69,8 @@ protected override FilterCriteria CreateFilterCriteria()
 
         protected override RoomSubScreen CreateRoomSubScreen(Room room) => new MultiplayerMatchSubScreen(room);
 
+        protected override ListingPollingComponent CreatePollingComponent() => listingPollingComponent = new MultiplayerListingPollingComponent();
```

Comment:
This is a little weird, can the polling component be exposed as a protected reference on the base class rather than doing this local write?

Reply:
Sure, I'm personally not too fussed either way but I understand the reasoning.